package.json

```diff
@@ -18,7 +18,7 @@
     "@supabase/auth-ui-react": "^0.4.2",
     "@supabase/auth-ui-shared": "^0.1.6",
     "@supabase/supabase-js": "^2.26.0",
-    "@superflows/chat-ui-react": "^1.1.6",
+    "@superflows/chat-ui-react": "^1.1.6-beta.1",
```

Comment:
what's the plan for release here? we have a beta on npmjs?

Reply:
Pushing a beta allows us to test in staging :smile: 

Like usual, `chat-ui` needs to be merged and published (to `1.1.7`) and this updated before merging